pkg/rke/k8s_version_info.go

```diff
@@ -47,16 +47,23 @@ func loadRancherDefaultK8sVersions() map[string]string {
 		"2.7.4":        "v1.25.x",
 		"2.7.5":        "v1.26.x",
 		"2.7.6":        "v1.26.x",
+		"2.7.7":        "v1.26.x",
+		"2.7.8":        "v1.26.x",
+		"2.7.9":        "v1.26.x",
```

Comment:
2.7.10 has been released and needs to be added to the list even though the branch will not be used in the Rancher v2.7.x line.

Reply:
Updated